custom_components/ramses_cc/__init__.py

```diff
@@ -241,67 +219,34 @@ def update_ha_state(self, delay=3) -> None:
         )  # HACK: call_soon_threadsafe should not be needed
 
 
-class RamsesDeviceBase(RamsesEntity):  # for: binary_sensor & sensor
-    """Base for any RAMSES II-compatible entity (e.g. BinarySensor, Sensor)."""
+class RamsesSensorBase(RamsesEntity):
+    """Base for any Ramses sensor entity."""
 
     def __init__(
         self,
         broker,
-        device,
+        rf_entity: RamsesRFEntity,
```

Comment:
I think we can leave this as `device`. The word `entity` is very HA, and so I avoid using that term if I can.